### DISCUSSION TITLE
Release v1

### DISCUSSION THREAD

**Discussion by jnath** [Novage/p2p-media-loader#332] [Q&A]
Hello, 

is there a release date for v1 branch ?

or maybe an alternative solution to be able to use v1 through npm.

mayby a scope is usefull for monorepo in v1 branche ?

`@p2p-media-loader/core`
`@p2p-media-loader/hlsjs`
...

or a simply beta tag in npm

**Comment by jnath** [comment]
i can create a MR if you want for the scope and publish in npm also. tell me.

**Comment by ElonJTrump** [comment]
I think if you can continue maintenance this rep you can fork it.

**Comment by mrlika** [comment]
Hi all,

v1 is in the latest development stage. It is a full rewrite to fix all the design issues we faced in v0:

- it supports the latest HLS v1 and Shaka v4
- it works on the latest iOS
- there is a Flutter demo that works in iOS, iPadOS, and Android

See https://github.com/Novage/p2p-media-loader/tree/v1

**Reply by ElonJTrump** [reply]
Hi, Mrlika

I'm very glad to see you update. My friend set up a company in Australia (Perth) and he want to fund this project. I tried to contact you from email you provided but never get back. 

Regards,
Steven

**Reply by mrlika** [reply]
Hi @stevenking888,

I appreciate your interest. It was a period when we were overwhelmed with commercial projects and unfortunately missed some opportunities with the P2P open-source project.

But now we have allocated resources to continue the development and are open to cooperation.

Please feel free to email us.

**Comment by mrlika** [comment]
1.0.0 was just released
